test/dummy/app/models/kitchen_sink.rb

```diff
@@ -7,6 +7,7 @@ class KitchenSink
   attribute :collection_radio_buttons_value, :boolean
   attribute :phone_field_value, :string
   attribute :email_field_value, :string
+  attribute :collection_check_boxes_value, :boolean
```

Comment:
This will be an array not a Boolean as it is a collection of check boxes. Don't worry for now though